src/inspect_ai/_eval/task/run.py

```diff
@@ -536,6 +553,8 @@ def handle_error(ex: BaseException) -> EvalError:
                                     ScoreEvent(score=score_result, target=sample.target)
                                 )
                                 results[scorer_name] = sample_score
+                # increment progress for scoring
```

Comment:
I kind of thought that we'd tick off progress before scoring? i.e. init is some "work" and counts for some, then the solvers run and count for nothing, then the scorer counts (but it's literally about to also tick off the sample so it might as well just be merged w/ the sample? Also, the scoring tick doesn't happen if cancellation or error occurs so it's lost.

I would argue we should probably just get rid of it b/c it's not actually providing unique progress and it can be missed. If we want to have something that runs _before_ scoring (representing the fact that solvers are done) I think we could do that (but note we need to be sure it is always ticked)

Reply:
Agree on all